cmd/ttn-lw-cli/commands/end_devices_split.go

```diff
@@ -375,3 +376,24 @@ func updateDeviceLocation(device *ttnpb.EndDevice, flags *pflag.FlagSet) {
 	}
 	device.Locations["user"] = loc
 }
+
+func parsePayloadFormatters(fmt *ttnpb.MessagePayloadFormatters) error {
```

Comment:
It feels a bit weird to put this in `_split.go`, but since we already have other stuff (that isn't splitting end devices) in here, I'll allow it.

Reply:
Moved to `utils.go`